lib/UserManagement.php

```diff
@@ -239,6 +241,9 @@ public function createOrganizationMembership($userId, $organizationId)
             $params,
             true
         );
+        if (!is_null($roleSlug) && !is_string($roleSlug)) {
+            throw new \InvalidArgumentException("roleSlug must be a string or null");
+        }
```

Comment:
We don't generally do runtime type checks for parameters like this. The typehint phpdoc comment should be enough.